src/Datadog.Trace.ClrProfiler.Managed/ScopeDBFactory.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Data;
+using Datadog.Trace.ClrProfiler.Integrations.AdoNet;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler
+{
+    internal static class ScopeDBFactory<T>
+    {
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(ScopeDBFactory<T>));
+
+        private static readonly string _fullName;
+        private static readonly string _dbTypeName;
+        private static readonly string _operationName;
+
+        static ScopeDBFactory()
+        {
+            var type = typeof(T);
+            _fullName = type.FullName;
+            _dbTypeName = ScopeFactory.GetDbType(type.Name);
+            _operationName = $"{_dbTypeName}.query";
+        }
+
+        public static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command)
+        {
+            if (_dbTypeName == null)
+            {
+                // don't create a scope, skip this trace
```

Comment:
I'm going to start fixing these `skip this trace` comments everywhere 😅 
```suggestion
                // don't create a scope, skip this span
```

Reply:
Haha